crates/astria-proto/src/native/sequencer.rs

```diff
@@ -0,0 +1,326 @@
+use std::{
+    error::Error,
+    fmt::Display,
+};
+
+use crate::generated::sequencer::v1alpha1;
+
+pub const ADDRESS_LEN: usize = 20;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct Address([u8; ADDRESS_LEN]);
+
+impl Address {
+    pub const LEN: usize = ADDRESS_LEN;
```

Comment:
```suggestion
    pub const LEN: usize = 20;
```

super nit but then you can remove `ADDRESS_LEN`?